src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -328,9 +341,9 @@ public enum Mode { STARTING, NORMAL, JOINING, LEAVING, DECOMMISSIONED, MOVING, D
     private Collection<Token> bootstrapTokens = null;
 
     // true when keeping strict consistency while bootstrapping
-    public static final boolean useStrictConsistency = Boolean.parseBoolean(System.getProperty("cassandra.consistent.rangemovement", "true"));
-    private static final boolean allowSimultaneousMoves = Boolean.parseBoolean(System.getProperty("cassandra.consistent.simultaneousmoves.allow","false"));
-    private static final boolean joinRing = Boolean.parseBoolean(System.getProperty("cassandra.join_ring", "true"));
+    public static final boolean useStrictConsistency = CONSISTENT_RANGEMOVEMENT.getBoolean();
```

Comment:
I would call these properties with `_` between the words even their keys are not with `_`. We can deal with this later to deprecate the old keys as it was done in DatabaseDescriptor.

Reply:
Fixed.